website/package.json

```diff
@@ -34,25 +33,25 @@
 		"@types/node": "^18.18.0",
 		"@types/react": "^18.2.23",
 		"@types/react-dom": "^18.2.8",
-		"@uiw/react-codemirror": "4.20.2",
+		"@uiw/react-codemirror": "4.21.21",
```

Comment:
The playground not working anymore is caused by this bump.

This version of `@uiw/react-codemirror` indirectly requires the version of `@codemirror/state` in range `^0.6.4` because of its dependencies: `@codemirror/view` and `@codemirror/commands`.

However [`@codemirror/state@6.2.1` is explicitly pinned in the `devDependencies`](https://github.com/biomejs/biome/blob/295f4bd4d46efe6b6927c2557b1b3292bd704806/website/package.json#L28), so two versions of `@codemirror/state` are installed and hence the [error occurs](https://github.com/uiwjs/react-codemirror/issues/216) which you can find in the console, and it breaks the playground.

A simple fix is to bump the `@codemirror/state` in `devDepedencies` to `0.6.4`.

Reply:
Nice catch, thanks @Sec-ant - hadn't noticed the dependencies were pinned, I've moved them to ranges instead and seem to be working fine locally!